src/gui/components/wxGameList.cpp

```diff
@@ -481,7 +478,7 @@ void wxGameList::OnContextMenu(wxContextMenuEvent& event)
 			menu.AppendSeparator();
 			menu.Append(kWikiPage, _("&Wiki page"));
 			menu.Append(kContextMenuGameFolder, _("&Game directory"));
-			menu.Append(kContextMenuSaveFolder, _("&Save directory"))->Enable(true);
+			menu.Append(kContextMenuSaveFolder, _("&Save directory"))->Enable(wxDirExists(gameInfo.GetSaveFolder().string()));
```

Comment:
Never call .string() on `std::filesystem::path`, it comes with all sorts of implementation-defined quirks. We have a small helper function for this, `_utf8Wrapper` which turns a path into a `std::string` with utf8 encoding. Which then can be turned into a wxString via helper `to_wxString`.

But in this case you can also just do `fs::exists(gameInfo.GetSaveFolder())` (fs is our alias for std::filesystem). But even this comes with a caveat. Under special (platform-specific) circumstances it can throw an exception which has to be handled. This can be avoided by providing the optional error code parameter and then instead of throwing it will return false. The robust solution would be:
```
std::error_code ec;
menu.Append(kContextMenuSaveFolder, _("&Save directory"))->Enable(fs::exists(gameInfo.GetSaveFolder(), ec));
```

Reply:
Right, I totally forgot that windows relies heavily on wchar instead of UTF-8.